ThreeDSecure/src/main/java/com/braintreepayments/api/ThreeDSecureRequest.java

```diff
@@ -217,6 +237,31 @@ public void setV1UiCustomization(@Nullable ThreeDSecureV1UiCustomization v1UiCus
         this.v1UiCustomization = v1UiCustomization;
     }
 
+    /**
+     * Optional. Sets all UI types that the device supports for displaying specific challenge user interfaces in the 3D Secure challenge.
+     * Possible Values:
+     * 01 BOTH
+     * 02 Native
+     * 03 HTML
+     *
+     * @param uiType {@link ThreeDSecureUiType} The UI type to request.
+     */
+    public void setUiType(@Nullable @ThreeDSecureUiType int uiType) {
```

Comment:
Should we specify that the default is BOTH?

Reply:
Good call, done: https://github.com/braintree/braintree_android/pull/738/commits/7f414880a38aad233d2446ed6f1d4892a190d3b2